.playwright/screenshots/snapshots.test.ts-snapshots/primer/alpha/action_bar/default/default.yml

```diff
@@ -0,0 +1,63 @@
+- toolbar:
```

Comment:
Is it useful or possible to throw these snapshots ino axe testing?

Reply:
Essentiall I am wondering if we can use these snapshots to validate that the markup is correct ... This might be a pointless question if we are already running axe on lookbook.

Reply:
We already have [Axe tests](https://github.com/primer/view_components/blob/main/test/accessibility_test.rb) running against these previews and in interactions that take place in [system tests](https://github.com/primer/view_components/tree/main/test/system/alpha)  🎉 .